md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ext/DataGenAPI.java

```diff
@@ -46,15 +46,14 @@
 @RequestMapping("/datagenproperties")
 public class DataGenAPI extends MetadataAPIBase {
     private static final Logger LOGGER = Logger.getLogger(DataGenAPI.class);
-    @Autowired
-    private PropertiesDAO propertiesDAO;
+    private static final String TABLE = "table";
     @Autowired
     private ProcessDAO processDAO;
 
     @RequestMapping(value = {"/createjobs"}, method = RequestMethod.POST)
 
-    public
-    @ResponseBody
+
+    @ResponseBody public
     RestWrapper createJobs(@RequestParam Map<String, String> map, Principal principal) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "createJobs" is 18 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "principal". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1172)